guides/integrate-nextauth.md

```diff
@@ -0,0 +1,327 @@
+---
+title: NextAuth
+sidebar_position: 6
+---
+
+import SetupJavaScript from '/docs/workflows/_sdk-setup/_setup-javascript.mdx';
+import Arcade, {Clip} from '/src/components/Arcade.tsx';
+import AppSchemeCaution from '/docs/workflows/\_app-scheme-caution.mdx';
+import InvocationTip from '/docs/workflows/\_invocation-type-tip.mdx';
+import InvocationDiagram from '/docs/platform-overview/\_invocation-url-diagram.mdx';
+
+# Integrate Beyond Identity Passwordless Authentication into NextAuth
+
+This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for a Next application that uses NextAuth.
```

Comment:
They are included a couple lines below, at the top of the "NextAuth" section. Should I move them up here?

Reply:
I like the link below. I was thinking linking to NextJS https://nextjs.org and then NextAuth's main page up here. 

Though I wonder if the links will change for NextAuth with the Auth.js?

Reply:
Oh, I misunderstood NextJS as NextAuth.js. I presumed you meant the repo and the website or something. 🙂 Yes, I can add a link to NextJS as well.

Yes, the links for NextAuth.js will change. Their repo already points to https://authjs.dev/ instead of https://next-auth.js.org/, though I think Auth.js is technically still pre-release.

Reply:
Done.